react-app/src/components/RHF/SlotField.tsx

```diff
@@ -242,13 +242,15 @@ export const SlotField = ({
         <div className={props?.wrapperClassName}>
           {fields?.map((S, i) => {
             return (
-              <FormField
-                key={`wrappedSlot-${i}`}
-                control={control}
-                name={parentId + S.name}
-                rules={ruleGenerator(S.rules, S.addtnlRules)}
-                render={RHFSlot({ ...S, control, parentId })}
-              />
+              <DependencyWrapper {...S}>
+                <FormField
+                  key={`wrappedSlot-${i}`}
+                  control={control}
+                  name={parentId + S.name}
+                  rules={ruleGenerator(S.rules, S.addtnlRules)}
+                  render={RHFSlot({ ...S, control, parentId })}
+                />
+              </DependencyWrapper>
```

Comment:
I'm not sure I understand. At yesterday's meeting @daniel-belcher suggested this would be the way to allow dependencies to work within nested checkboxes. If you think `FieldArray` is a better way to handle it then sure, we can discuss at stand-up today

Reply:
Just an update for posterity: During stand-up, we decided that this approach would be best because of the psudo-child way that we render the Textarea as a dependency of the Select component.